projects/mercury/src/metadata/views/MetadataView.styles.js

```diff
@@ -17,6 +17,13 @@ const styles = theme => ({
         maxHeight: 'calc(100vh - 210px)',
         overflowY: 'auto'
     },
+    facetHeaders: {
+        textAlign: 'left',
+        marginTop: 6,
+        fontSize: 13,
+        color: '#3f51b5',
```

Comment:
For consistency with the other colours:
```suggestion
        color: theme.palette.primary.light,
```